api/desecapi/serializers/tokens.py

```diff
@@ -52,6 +53,7 @@ class Meta:
             "id",
             "created",
             "last_used",
+            "mfa",
```

Comment:
add a test for non-writeability. Since this field is high-stakes and different type (i.e., perhaps different code path) from the tested ones, I think an additional test is warranted.